docs/_config.yml

```diff
@@ -14,12 +14,7 @@ ALLUXIO_MASTER_VERSION_SHORT: master
 ALLUXIO_MAJOR_VERSION: master
 # We must inline the version string (e.g., "1.4.0-SNAPSHOT") rather than using the macro of Alluxio version.
 # Otherwise the macro name remains in the output.
-ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/default/alluxio-1.7.0-SNAPSHOT-default-client.jar
-ALLUXIO_CLIENT_JAR_PATH_BUILD: /<PATH_TO_ALLUXIO>/core/client/runtime/target/alluxio-core-client-runtime-1.7.0-SNAPSHOT-jar-with-dependencies.jar
-ALLUXIO_CLIENT_JAR_PATH_FLINK: /<PATH_TO_ALLUXIO>/client/flink/alluxio-1.7.0-SNAPSHOT-flink-client.jar
-ALLUXIO_CLIENT_JAR_PATH_HADOOP: /<PATH_TO_ALLUXIO>/client/hadoop/alluxio-1.7.0-SNAPSHOT-hadoop-client.jar
-ALLUXIO_CLIENT_JAR_PATH_PRESTO: /<PATH_TO_ALLUXIO>/client/presto/alluxio-1.7.0-SNAPSHOT-presto-client.jar
-ALLUXIO_CLIENT_JAR_PATH_SPARK: /<PATH_TO_ALLUXIO>/client/spark/alluxio-1.7.0-SNAPSHOT-spark-client.jar
+ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/alluxio-1.7.0-SNAPSHOT-client.jar
```

Comment:
Do we still want to keep this path, or just direct to the `core/client/runtime` path?

Reply:
discussed offline. `core/client/runtime/target` is too deep and not proper to stay in the final release tarball. I will keep the top level `client/` dir and add it to `.gitignore`